src/brain-gain/main.css

```diff
@@ -95,6 +95,26 @@ core-navbar > div {
   transition: 0.6s;
 }
 
+.game-details-title{
```

Comment:
nit pick: add a space before bracket for consistency (don't forget the other ones below)